src/Sarif/Autogenerated/SarifRewritingVisitor.cs

```diff
@@ -330,6 +330,14 @@ public virtual ExternalPropertyFiles VisitExternalPropertyFiles(ExternalProperty
                     }
                 }
 
+                if (node.ThreadFlowLocations != null)
+                {
+                    for (int index_0 = 0; index_0 < node.ThreadFlowLocations.Count; ++index_0)
+                    {
+                        node.ThreadFlowLocations[index_0] = VisitNullChecked(node.ThreadFlowLocations[index_0]);
+                    }
+                }
+
```

Comment:
this is strictly not part of this PR, but when i regenerated this file i noticed this additional check and thought of bringing is to attention in case it was mistakenly not included earlier?